src/components/RepoCard/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { AiOutlineStar } from 'react-icons/ai';
+import { VscRepoForked } from 'react-icons/vsc';
+import { TbCircleDot } from 'react-icons/tb';
+
+import styles from './index.module.css';
+
+const RepoCard = ({ repo }) => {
+    const {
+        name,
+        description,
+        stargazers_count,
+        open_issues_count,
+        forks_count,
+
+        created_at,
+        owner,
+    } = repo;
+    console.log(repo);
+
+    return (
+        <div className={styles.card}>
+            <div className={styles.firstSection}>
+                <a
```

Comment:
In the Header Component + RepoCard, your a tags could be Link tags 
(I believe they get converted to a tags within the react bundler 
and so would keep the dynamic styles.a classname)